src/components/Map.js

```diff
@@ -18,12 +54,20 @@ function Map(props) {
         url="https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}"
         attribution="Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community"
       />
-      <Marker position={props.center} />
+      <Marker position={location} />
       <Polygon
         positions={vtOutline}
-        pathOptions={{ color: "orange", fillOpacity: 0 }}
-      />
+        pathOptions={{ color: "orange", fillOpacity: 0 }}></Polygon>
+       <ChangeView center={props.location} zoom={ZoomIn} location={location} /> 
+        <ReturnButton center={props.location} zoom={ZoomIn} location={location} ReturnLocation={ReturnLocation}/>
+      <MoveView center={props.location} zoom={ZoomIn} location={location} MoveLocation={MoveLocation} setMoveLocation={setMoveLocation} startPress={props.startPress} />
```

Comment:
It looks like you're importing two different components into your Map that control where you page is zoomed in on, This is a bit redundant and can all be managed through one section of code